pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -169,8 +169,8 @@ if [ -n "$NIX_DEBUG" ]; then
   done
 fi
 
-if [ -n "$NIX_CC_WRAPPER_EXEC_HOOK" ]; then
-    source "$NIX_CC_WRAPPER_EXEC_HOOK"
+if [ -n "$NIX_@infixSalt@_CC_WRAPPER_EXEC_HOOK" ]; then
+    source "$NIX_@infixSalt@_CC_WRAPPER_EXEC_HOOK"
 fi
 
 PATH="$path_backup"
```

Comment:
I probably misguided you a couple of times by clicking the general area as to where the problem was. I meant.  `PATH="$path_backup"`